openshift_cli_installer/libs/clusters/ocm_cluster.py

```diff
@@ -42,18 +44,29 @@ def _set_expiration_time(self):
             )
 
     def get_osd_versions(self):
-        self.osd_base_available_versions_dict.update(
-            Versions(client=self.ocm_client).get(channel_group=self.cluster_info["channel-group"])
-        )
+        updated_versions_dict: Dict[str, Dict[str, List[str]]] = {}
+        for cannel, versions in (
+            Versions(client=self.ocm_client).get(channel_group=self.cluster_info["channel-group"]).items()
+        ):
+            updated_versions_dict[cannel] = {}
```

Comment:
missing typing

Reply:
No, you can't add typing to '{}['something']'